### PR TITLE
Bump software requirements for final benchmarking run

### PR DESCRIPTION
**Pull request by sbesson**
- Bump OME Files to 0.3.1
- Bump Bio-Formats to 5.3.4
- Bump Bio-Formats JACE to 5.3.1

--depends-on https://github.com/openmicroscopy/ome-files-cpp-u1604-docker/pull/6
--depends-on https://github.com/ome/bio-formats-jace/pull/9

/cc @rleigh-codelibre

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -68,11 +68,11 @@ include(BoostChecks)
...
 find_package(XercesC)
 find_package(XalanC)
-find_package(OMECompat 5.2.0 REQUIRED)
-find_package(OMECommon 5.2.0 REQUIRED)
-find_package(OMEXML 5.2.0 REQUIRED)
-find_package(OMEFiles 0.2.3 REQUIRED)
-find_package(BioFormatsJACE 5.3.0)
+find_package(OMECompat 5.4.0 REQUIRED)
+find_package(OMECommon 5.4.0 REQUIRED)
+find_package(OMEXML 5.5.0 REQUIRED)
```
Comment (rleigh-codelibre) [inline CMakeLists.txt]: 5.5.1 to match files 0.3.1?

Comment (sbesson) [reply CMakeLists.txt]: Oops. Absolutely

**Comment by rleigh-codelibre** [comment]
All the versions here look fine, though it will need the other PRs merging to fully test

**Comment by rleigh-codelibre** [comment]
Windows builds green, looks good to merge.

**Comment by rleigh-codelibre** [comment]
I've had a small look over the results, they look sensible; could we merge this so we can run the analysis?

**Comment by sbesson** [comment]
Thanks for double checking, merging

**Pull request merged**